frontend/src/views/CensusDirectory/index.tsx

```diff
@@ -0,0 +1,262 @@
+import {
```

Comment:
let's clean up this file by following Ronen's new file structure please!

Example: https://github.com/chanzuckerberg/single-cell-data-portal/blob/b92b661461b2afcfbbc707de412f50c5b1b979cd/frontend/src/views/WheresMyGeneV2/components/Filters

Reply:
will do before feature launch!